docs/openj9_signals.md

```diff
@@ -0,0 +1,174 @@
+<!--
+* Copyright (c) 2017, 2020 IBM Corp. and others
```

Comment:
copyright: year(file added), year(file updated). since this is a completely new file, year(file added) = 2020.
```suggestion
* Copyright (c) 2020, 2020 IBM Corp. and others
```

Reply:
Docs are treated as one whole, so copyright is consistent across the doc set. 2017, 2020 accurate